discovery-provider/src/tasks/index_aggregate_user.py

```diff
@@ -362,14 +366,20 @@ def update_aggregate_table(
 
             with db.scoped_session() as session:
                 latest_indexed_block_num = get_latest_blocknumber(session, redis)
-                start_time = time.time()
+
                 if (
                     not most_recent_indexed_aggregate_block
-                    or latest_indexed_block_num % REFRESH_COUNTER == 0
                 ):
-                    # re-create entire table
+                    # repopulate entire table
+                    logger.info(f"index_aggregate_user.py | Repopulating {table_name}")
                     most_recent_indexed_aggregate_block = 0
                     session.execute("TRUNCATE TABLE {}".format(table_name))
+                elif latest_indexed_block_num % REFRESH_COUNTER == 0:
```

Comment:
I'm not liking this method anymore. Gonna change this to use a last completed timestamp saved in redis, then update based on that.